bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -74,9 +74,11 @@ void ReplicaImp::registerStatusHandlers() {
 
```

Comment:
Couldn't we extract all this concord-ctl related code to a different class and have it as a data member in ReplicaImp?